deploy/helm/grafana-operator/templates/servicemonitor.yaml

```
@@ -29,11 +29,11 @@ spec:
       {{- end }}
       {{- if .Values.serviceMonitor.metricRelabelings }}
       metricRelabelings:
-      {{ toYaml .Values.serviceMonitor.metricRelabelings | indent 4 }}
+      {{- toYaml .Values.serviceMonitor.metricRelabelings | indent 6 }}
       {{- end }}
       {{- if .Values.serviceMonitor.relabelings }}
       relabelings:
-      {{ toYaml .Values.serviceMonitor.relabelings | nindent 4 }}
+      {{- toYaml .Values.serviceMonitor.relabelings | nindent 6 }}
```

```suggestion
      {{- toYaml .Values.serviceMonitor.relabelings | nindent 8 }}
```

It's worth increasing the indentation here as well.